src/backend/Csrs.Api/Models/ModelExtensions.cs

```diff
@@ -130,6 +130,9 @@ public static MicrosoftDynamicsCRMssgCsrsfile ToDynamicsModel(this File file)
 
             MicrosoftDynamicsCRMssgCsrsfile dynamicsFile = new MicrosoftDynamicsCRMssgCsrsfile
             {
+                Statecode = 0,
```

Comment:
put some comments on what these codes mean... ideally we would have some constants. Someone not familiar doesnt know what these 0 and 1s mean